packages/pools/src/constants/pools/97.ts

```diff
@@ -4,15 +4,15 @@ import { getAddress } from 'viem'
 import { PoolCategory, SerializedPool } from '../../types'
 
 export const livePools: SerializedPool[] = [
-  // {
-  //   sousId: 0,
-  //   stakingToken: bscTestnetTokens.cake2,
-  //   earningToken: bscTestnetTokens.cake2,
-  //   contractAddress: '0x4c650FB471fe4e0f476fD3437C3411B1122c4e3B',
-  //   poolCategory: PoolCategory.CORE,
-  //   tokenPerBlock: '0.01',
-  //   isFinished: false,
-  // },
+  {
+    sousId: 0,
+    stakingToken: bscTestnetTokens.cake2,
+    earningToken: bscTestnetTokens.cake2,
+    contractAddress: '0x4c650FB471fe4e0f476fD3437C3411B1122c4e3B',
```

Comment:
@Chef-Yogi this address is not a valid pool, test will failed. I'll comment it back firstly, let me known if have some update